src/client.rs

```diff
@@ -1079,15 +1079,28 @@ impl Spotify {
     ///
     /// Parameters:
     /// - limit - the number of entities to return
+    /// - time_limit - A Unix timestamp in milliseconds.
     ///
     /// [Reference](https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-the-users-currently-playing-track)
     #[maybe_async]
-    pub async fn current_user_recently_played<L: Into<Option<u32>>>(
+    pub async fn current_user_recently_played<L: Into<Option<u32>>, T: Into<Option<TimeLimits>>>(
         &self,
         limit: L,
+        time_limit: T,
     ) -> ClientResult<CursorBasedPage<PlayHistory>> {
-        let mut params = Query::with_capacity(1);
+        let mut params = Query::with_capacity(2);
         params.insert("limit".to_owned(), limit.into().unwrap_or(50).to_string());
+        if let Some(l) = time_limit.into() {
+            match l {
+                TimeLimits::Before(y) => {
+                    params.insert("before".to_owned(), y.to_string());
+                }
+                TimeLimits::After(y) => {
+                    params.insert("after".to_owned(), y.to_string());
+                }
+            };
+        }
+
         let result = self
             .endpoint_get("me/player/recently-played", &params)
             .await?;
```

Comment:
I suppose that you are using Vim to edit files, since you are commiting the swap file `src/model/enums/.misc.rs.swp ` to this repository. It's useful for you, but not for us :) (just a joke)